src/app/service/data-service.service.ts

```diff
@@ -0,0 +1,58 @@
+import { Injectable } from '@angular/core';
+import { Item, List } from '../model';
+
+@Injectable({
+  providedIn: 'root',
+})
+export class DataService {
+  private data: List[] = [
+      {
+        id: 'aaa',
+        name: 'list_1',
+        item: [
+          {
+            list: 'list_1',
+            id: 'list_id_1',
+            title: 'Title list_1',
+            description: 'this is the descr for Titile list_1',
+            completed: false,
+          },
+        ],
+      },
+      {
+        id: 'bbb',
+        name: 'list_2',
+        item: [
+          {
+            list: 'list_2',
+            id: 'list_id_2',
+            title: 'Title list_2',
+            description: 'this is the descr for Titile list_2',
+            completed: false,
+          },
+        ],
+      },
+
+  ];
+
+  listData() {
+    return this.data;
+  }
+  // Add an item in a specific list
+  addItem(idList: string, todo: Item) {
```

Comment:
make this function able to add more than one item (use variadic parameter for `todo`)